pkg/drivers/kvm/kvm.go

```diff
@@ -371,7 +371,7 @@ func (d *Driver) Remove() error {
 	}
 
 	log.Debug("Checking if the domain needs to be deleted")
-	dom, err := conn.LookupDomainByName(d.MachineName)
+	dom, _ := conn.LookupDomainByName(d.MachineName)
```

Comment:
it's probably still worth checking the error @r2d4

Reply:
I'm now checking both of these errors.  I added a logged warning if they don't exist, since this is just the cleanup function so I don't think we care to throw any errors here if there isn't anything to clean up for some reason.

Reply:
got it, ok :+1: